main.py

```diff
@@ -15,21 +16,29 @@
 from containers import LiveStreamContainer
 from videoframegrid import VideoFrameGrid
 from config import cfg
+from add_stream_error import AddStreamError
```

Comment:
This would then become `enums` which perhaps would be a bit more declarative as well?